src/index.js

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { init, locations } from 'contentful-ui-extensions-sdk';
+import * as Sentry from '@sentry/react';
 import App from './extensions/App';
 import mockLocations from './__mocks__/mockLocations';
 
 (()=> {
+  Sentry.init({dsn: "https://745cf65377674adbaf85942851bbb125@o267347.ingest.sentry.io/5320754"});
```

Comment:
Move this whole URL into an env file.   Might need to update the webpack build.   Google "using an env in a create react app."   Ex: https://create-react-app.dev/docs/adding-custom-environment-variables/